### PR TITLE
Add issue tracker integration with GitHub and Clubhouse

### PR DESCRIPTION
**Pull request by alex-tessian**
Addresses #27

### CODE REVIEW

**Comment by alex-tessian** [comment]
Actually I think some of this depends on some changes in #31

File: catapult/integrations/clubhouse.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+)
+
+
+class Clubhouse(BaseTracker):
+    def get_linked_prs(self, issue_id: str) -> List[PullRequest]:
+        git_provider = get_git()
+        if issue_id.startswith("ch"):
+            issue_id = issue_id[2:]
+        url = CH_STORY_ENDPOINT.format(
+            story_id=issue_id, ch_token=os.environ["CH_TOKEN"]
```
Comment (fede-tessian) [inline catapult/integrations/clubhouse.py:27]: can you add the new env vars to the README pls?

File: catapult/tickets.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Commands to interact with issue trackers
+"""
+from typing import List, Mapping
+
+import invoke
+
+from catapult import utils
+from catapult.integrations.git import PullRequestState
+from catapult.integrations.tracker import get_tracker
+from catapult.projects import format_projects, list_projects
```
Comment (fede-tessian) [inline catapult/tickets.py:11]: I'm getting an import error for `format_projects` in this line when running `catapult tickets`

File: catapult/integrations/clubhouse.py (hunk 1 above)
Comment (fede-tessian) [reply catapult/integrations/clubhouse.py:27]: _Nice to have: a link or short description how to get those values from github and clubhouse

File: catapult/tickets.py (hunk 2 above)
Comment (alex-tessian) [reply catapult/tickets.py:11]: This is because this PR depends on some changes in #31. If you merge with that it should run ok.

Comment (alex-tessian) [reply catapult/tickets.py:11]: (Sorry it was hard to pull them apart as I'd been working on them both concurrently)

**Pull request merged**